ethereum_bridge/src/protocol/transactions/validator_set_update/mod.rs

```diff
@@ -141,3 +152,86 @@ where
 
     Ok(changed)
 }
+
+#[cfg(test)]
+mod test_valset_upd_state_changes {
```

Comment:
I know its more difficult, but I would love a test when there are votes but not a quorum.